src/ServiceInsight/Saga/SagaWindowViewModel.cs

```diff
@@ -99,6 +106,12 @@ string GetMajorVersion(StoredMessage message)
             return version?.Split('.').First();
         }
 
+        void ClearSaga()
+        {
+            Data = null;
+            NotifyOfPropertyChange(() => Data);
```

Comment:
This line is redundant. Setting the property will notify of the change anyway because of PropertyChanged.Fody.